shared/js/background/utils.es6.js

```diff
@@ -38,6 +38,16 @@ export async function sendTabMessage (id, message, details) {
     }
 }
 
+export async function sendAllTabsMessage (message, details) {
```

Comment:
This is great, we should migrate the logout and ddgUserReady to this; can you file a follow up task?

Reply:
Good idea, [here you go](https://app.asana.com/0/892838074342800/1203234753874834/f).